cpg_workflows/jobs/joint_genotyping.py

```diff
@@ -92,11 +106,12 @@ def make_joint_genotyping_jobs(
         with sample_map_bucket_path.open('w') as fp:
             df.to_csv(fp, index=False, header=False, sep='\t')
 
+    # For small callsets, we don't apply the ExcessHet filtering anyway
     do_filter_excesshet = len(gvcf_by_sgid) >= 1000 and do_filter_excesshet
```

Comment:
this line and comment was moved from a few lines down. We did
```do_filter_excesshet = len(gvcf_by_sgid) >= 1000 and do_filter_excesshet``` 
here, then later did a boolean test for 
```if len(gvcf_by_sgid) >= 1000 and do_filter_excesshet:```
which feels silly